cmake/settings/ini.cmake

```diff
@@ -0,0 +1,113 @@
+####
+# ini.cmake:
+#
+# This file loads settings from INI files. In cases where these settings are supplied via fprime-util, the settings are
+# checked for discrepancy. This allows the cmake system to detect when it should be regenerated and has not been. In the
+# case that the settings were not supplied, it sets them.
+####
+include_guard()
+
+# Necessary global variables
+set(SETTINGS_CMAKE_DIRECTORY "${CMAKE_CURRENT_LIST_DIR}")
+find_program(PYTHON NAMES python3 python) #This happens before required
+
+####
+# FPRIME_UTIL_CRITICAL_LIST:
+#
+# This is a list of critical settings that are passed in from fprime-util. If these settings change in `settings.ini`
+# they need to result in a WARNING to let the user know that had fprime-util be run, it now needs to be rerun.
+####
+set(FPRIME_UTIL_CRITICAL_LIST
+    "FPRIME_FRAMEWORK_PATH"
+    "FPRIME_LIBRARY_LOCATIONS"
+    "FPRIME_PROJECT_ROOT"
+    "FPRIME_ENVIRONMENT_FILE"
+    "FPRIME_AC_CONSTANTS_FILE"
+    "FPRIME_CONFIG_DIR"
+    "FPRIME_INSTALL_DEST"
+)
+
+####
+# init_to_cache:
```

Comment:
Typo here that may be worth fixing if this generates docs somewhere.

Reply:
Fixed.